### DISCUSSION TITLE
dns劫持模式那个性能更好

### DISCUSSION THREAD

**Discussion by linyueh** [vernesong/OpenClash#3006] [Q&A]
看到当前版本dns劫持模式可选择dnsmasq转发和防火墙转发，请问两种模式有什么区别，哪个性能更好

**Comment by TnZzZHlp** [comment]
防火墙转发少走一个dnsmasq，延迟会低一点点，实际使用没有区别
但是部分功能依赖dnsmasq，所以有需求的就还是老老实实用dnsmasq转发
